build.sh

```diff
@@ -257,6 +273,14 @@ if [ ${BUILD_BE} -eq 1 ] ; then
             -DGLIBC_COMPATIBILITY=${GLIBC_COMPATIBILITY} ../
     ${BUILD_SYSTEM} -j ${PARALLEL}
     ${BUILD_SYSTEM} install
+    echo "Build Frontend Modules: java-udf"
+    cd ${DORIS_HOME}/fe
+    if [ ${CLEAN} -eq 1 ]; then
+        clean_fe
+    fi
+    if [ ${BUILD_FE} -eq 0 ]; then
+      ${MVN_CMD} package -pl fe-common,fe-core,java-udf -DskipTests
+    fi
```

Comment:
why build fe after build be?

Reply:
@yangzhg The order is not important and I just keep the origin order.

Reply:
@Gabriel39 Could you modify the build.sh to just build be only when execute `build.sh --be`？

Reply:
@adonis0147 pls refer to https://github.com/apache/incubator-doris/pull/8612/files